scripts/check-missing.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+GOVCMS_BUILD="${GOVCMS_BUILD:-/tmp/satis-build-conflict-check}"
+rm -Rf "${GOVCMS_BUILD}"
+
+SATIS_BUILD="${SATIS_BUILD:-./app}"
+php -S localhost:4142 -t "${SATIS_BUILD}" > /tmp/phpd.log 2>&1 &
+
+# Install raw govcms that points to the latest releases.
+composer create-project --no-install govcms/govcms8-scaffold-paas "${GOVCMS_BUILD}"
+cd "${GOVCMS_BUILD}"
+cp composer.json composer-copy.json && cat composer-copy.json \
+  | jq .repositories='{"drupal":{"type":"composer","url": "https://packages.drupal.org/8"},"custom":{"type":"path","url":"custom/composer"},"govcms":{"type":"composer","url":"http://localhost:4142"}}' \
+  | tee composer.json > /dev/null
+COMPOSER_MEMORY_LIMIT=-1 composer update
+
+# Get a list of desired versions from the world.
+target_list=$(composer info --format=json | jq -r '.installed[] | "\(.name) \(.version)"' | sort)
+
+# Point to the local satis build.
+composer config secure-http false
```

Comment:
I've had to move this before the composer update in L17, otherwise it won't work?

Reply:
I agree, I don't know how it was working like that